src/app/common/templates/job-body.html

```diff
@@ -30,7 +37,7 @@
             <li class="job-description">{{ job.attributes.description }}</li>
         </ul>
         <ul ng-if="job.attributes.description_translation.from != job.attributes.description_translation.to">
-            <li class="job-description translated" ng-show="dt.job_item">
+            <li class="job-description" ng-show="dt.job_item">
```

Comment:
It doesn't have effect on the translation. That was just a css class which is not defined and so it is not used at all.